modules/custom/activity_creator/src/Service/ActivityCreatorBatchActivityDeletion.php

```diff
@@ -47,7 +48,7 @@ public static function updateProcess(array $items, array &$context): void {
     $activity_storage = \Drupal::entityTypeManager()->getStorage('activity');
 
     // Elements per operation.
```

Comment:
@nechai It's better to change the comment to:
`// Ensure the platform can determine how many items we process.`